src/styles/blocks/card.css

```diff
@@ -0,0 +1,109 @@
+.card {
+  position: relative;
+  width: 200px;
+  height: 406px;
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  border-radius: 5px;
+}
+
+.card__images {
+  position: absolute;
+  top: 33px;
+  left: 20px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__name {
+  position: absolute;
+  top: 206px;
+  left: 17px;
+  color: #060b35;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  width: 166px;

```

Comment:
do not hardcode width, set paddings to card to set right spacing and ad * {box-sizing: border-box;}
<img width="203" alt="image" src="https://user-images.githubusercontent.com/66749919/198832096-d2690d46-f4c2-4f93-a3cc-fbe092e20f22.png">